chain/rosetta-rpc/src/lib.rs

```diff
@@ -30,6 +30,12 @@ pub const BASE_PATH: &str = "";
 pub const API_VERSION: &str = "1.4.4";
 pub const BLOCKCHAIN: &str = "nearprotocol";
 
+/// Genesis together with genesis block identifier.
+struct GenesisWithIdentifier {
+    genesis: Genesis,
+    block_id: models::BlockIdentifier,
+}
```

Comment:
This looks like a start of RosettaCtx from https://github.com/near/nearcore/issues/6848